easybuild/easyconfigs/j/Java/Java-1.9.0.4.eb

```diff
@@ -0,0 +1,19 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+name = 'Java'
+version = '1.9.0.4'
+
+homepage = 'http://java.com/'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# download the tar.gz directly from
+# http://www.oracle.com/technetwork/java/javase/downloads/index.html
+altver = version.split('.', 1)[1]
+sources = ['jdk-%s_linux-x64_bin.tar.gz' % altver]
```

Comment:
@sassy-crick No need for `altver`, you can use the `%(version_minor)s` instead (see also https://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html#avail-easyconfig-templates)

```python
sources = ['jdk-%(version_minor)s_linux-x64_bin.tar.gz']

Reply:
@boegel Thanks. All I done was changing the already existing file to the latest version. Do you want me to change it the way you suggest?

Reply:
Hold on... This rings a bell.

If the tarball is named `jdk-9_*`, then how will we discriminate future 1.9.x versions from this one?!

Reply:
The tarball is currently called: jdk-9.0.4_linux-x64_bin.tar.gz and a previous one was called  jdk-9.0.1_linux-x64_bin.tar.gz. From what I can see on the Oracle home page, v. 1.9. is no longer actively developed and the latest development is v. 1.11. 
The module gets correctly assigned as Java/1.9.0.4 so that should work as well, as far as I can see it. I am sure I am missing out something here.

Reply:
@sassy-crick OK, so my proposal isn't going to work, basically. In fact, it doesn't make much sense at all, so ignore it. :)